cloudant-client/src/test/java/com/cloudant/tests/CloudantClientTests.java

```diff
@@ -136,7 +136,8 @@ public void testUserAgentHeaderIsAddedToRequest() throws Exception {
             //instantiating the client performs a single post request
             CloudantClient client = CloudantClientHelper.newMockWebServerClientBuilder(server)
                     .build();
-            client.executeRequest(createPost(client.getBaseUri(), null, "application/json"));
+            client.executeRequest(createPost(client.getBaseUri(), null, "application/json"))

```

Comment:
These sorts of invocations could trigger findbugs `RV_RETURN_VALUE_IGNORED_INFERRED` warnings.